ci-operator/step-registry/gather/audit-logs/gather-audit-logs-commands.sh

```diff
@@ -17,6 +17,12 @@ then
 	source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+if [[ -f ${SHARED_DIR}/BASTION_IP ]]; then
```

Comment:
I'll move that to a script, and do like baremetal is doing (see above) do just source it.